### PR TITLE
Chatkit: Android12 migration

### PR DESCRIPTION
**Pull request by rbenza**

### CODE REVIEW

File: chatkit/build.gradle [hunk 1]
```diff
@@ -17,15 +18,12 @@ android {
...
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
+    implementation fileTree(dir: "libs", include: ["*.jar"])
 
-    implementation 'androidx.appcompat:appcompat:1.2.0'
-    implementation 'com.google.android.material:material:1.0.0'
-    implementation "com.google.android.flexbox:flexbox:$flexboxVersion"
+    implementation "androidx.appcompat:appcompat:$appcompat_version"
+    implementation "com.google.android.material:material:$material_design_version"
+    implementation "com.google.android.flexbox:flexbox:$flexbox_version"
+    implementation 'androidx.legacy:legacy-support-v4:1.0.0'
```
Comment (dzolnai) [inline chatkit/build.gradle:28]: New lib, why is this needed?

**Review by dzolnai — CHANGES_REQUESTED** [review]

File: chatkit/build.gradle (hunk 1 above)
Comment (rbenza) [reply chatkit/build.gradle:28]: Build error, missing a view inside Chatkit SDK. Adding lib fixed it. Think it was a dep previously in older version of flexbox.

**Review by dzolnai — APPROVED** [review]

**Comment by Meghdut-Mandal** [comment]
How can I use the latest version of the chatkit library? The official repo is hardly maintained.

**Comment by dzolnai** [comment]
@Meghdut-Mandal you can add it as a submodule to your repo and then include it as a library module in Android Studio.
Please keep in mind that this is our own fork for our own purposes. We do not guarantee any support or compatibility with other apps.

**Comment by Meghdut-Mandal** [comment]
Thanks, @dzolnai  for the info. I would be using it for one of my personal projects so no issues with support.
